lib/livebook_cli/utils.ex

```diff
@@ -0,0 +1,34 @@
+defmodule LivebookCLI.Utils do
+  def option_parse(argv, opts \\ []) do
+    {parsed, argv, errors} = OptionParser.parse(argv, opts)
+    {Enum.into(parsed, %{}), argv, errors}
+  end
+
+  def log_info(message) do
+    IO.puts(message)
+  end
+
+  if Mix.env() == :dev do
+    def log_debug(message) do
+      [:cyan, message]
+      |> IO.ANSI.format()
+      |> IO.puts()
+    end
+  else
+    def log_debug(_message) do
+      :ok
+    end
+  end
+
+  def log_warning(message) do
+    [:yellow, message]
+    |> IO.ANSI.format()
+    |> IO.puts()
+  end
+
+  def print_text(message) do
```

Comment:
We potentially don't need `print_text` and `log_info` right? Perhaps we stick with `log_info` (but use the `print_text` implementation)?